package.json

```diff
@@ -171,19 +171,21 @@
     "jest-enzyme": "^2.1.0",
     "lost": "^8.0.0",
     "nightmare": "^2.7.0",
+    "nock": "^9.0.2",
     "nodemon": "^1.10.2",
     "postcss-browser-reporter": "^0.5.0",
     "postcss-cssnext": "^2.8.0",
     "postcss-font-magician": "^1.4.0",
     "postcss-loader": "^1.2.2",
-    "postcss-modules": "^0.6.3",
+    "postcss-modules": "^0.6.4",
```

Comment:
no and i don't think we are technically using this dep...i just do a `yarn upgrade-interactive` on every PR...that CLI command is available in 0.20.0.

I'll remove it another PR that address general dependency clean up.